packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -38,6 +38,9 @@ const allowedErrors = {
   NetworkError: 'NetworkError when attempting to fetch resource.',
   // Webkit browser error
   CORSError: 'Cross-origin redirection',
+
+  // prompt=none errors
+  InteractionNotAllowed: 'The request requires some interaction that is not allowed.',
```

Comment:
I'm a bit confused here. Being that the `parseError` method concatenates the `error` and `error_description`, I would think this string would start with `interaction_required: `: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/packages/javascript-sdk/src/oauth2-client/index.ts#L296. Am I misunderstanding what happens here?

Reply:
<img width="1237" alt="Screen Shot 2023-02-15 at 2 52 49 PM" src="https://user-images.githubusercontent.com/18267769/219177381-4d50b51a-6e6d-4f42-b823-f6c1e150a2e1.png">

Reply:
Ah, okay, you're right. Sorry for the confusion.